src/tools/ceph-dencoder/denc_plugin.h

```diff
@@ -66,6 +66,7 @@ class DencoderPlugin {
 };
 
 #define TYPE(t) plugin->emplace<DencoderImplNoFeature<t>>(#t, false, false);
+#define TYPE_VARARGS(t, ...) plugin->emplace<DencoderImplNoFeature<t>>(#t, false, false, ##__VA_ARGS__);
```

Comment:
Does this definition have any users?

Reply:
Not yet, but for future use with arguments